karma.conf.js

```diff
@@ -28,6 +30,30 @@ const newWebpackConfig = Object.assign({}, webpackConfigProd, {
     new webpack.NormalModuleReplacementPlugin(/core\/logger$/, 'core/client/logger.js'),
     // Use the browser's window for window.
     new webpack.NormalModuleReplacementPlugin(/core\/window/, 'core/browserWindow.js'),
+    // Plugin to show any webpack warnings and prevent tests from running
+    // Based on: https://gist.github.com/Stuk/6b574049435df532e905
+    function WebpackWarningPlugin() {
+      this.plugin('done', (_stats) => {
+        const stats = _stats;
+        let flagError;
+
+        if (stats.compilation.warnings.length) {
+          // Log each of the warnings
+          stats.compilation.warnings.forEach((_warning) => {
+            const warning = _warning.message || _warning || '';
+            if (warning.indexOf('SyntaxError') > -1) {
+              console.log(warning);
+              flagError = true;
```

Comment:
Unfortunately there appears to be some noise. E.g. the issue for linux was that webpack was reporting not being able to get at dtrace-provider even when it shouldn't even have cared since we sub-in the client logger for the karma tests. I suspect this is related to the way that webpack introspects the requires and doesn't actually mean there's a problem. 

I wouldn't be against us logging everything if you thought that was better though. Feel free to open a new issue or just simply move the line if that's quicker/easier.